scripts/scripts.js

```diff
@@ -428,7 +428,7 @@ export function decorateIcons(block = document) {
 async function waitForLCP() {
   // eslint-disable-next-line no-use-before-define
   const lcpBlocks = LCP_BLOCKS;
-  const block = document.querySelector('.block');
+  const block = document.querySelector('.block.form') || document.querySelector('.block');
```

Comment:
Will that also work if the form is at the end of the page?

Reply:
humm... correct. We'll lose the LCP block from the page and wait for the form at the bottom. But do we have this case somewhere ?